elastica/dissipation.py

```diff
@@ -151,3 +154,115 @@ def dampen_rates(self, rod: SystemType, time: float):
         rod.omega_collection[:] = rod.omega_collection * np.power(
             self.rotational_exponential_damping_coefficient, rod.dilatation
         )
+
+
+class FilterDamper(DamperBase):
```

Comment:
Lets use some more explicit name for the damper.

- `SecondOrderOperator` : Exact name used in the referenced paper.
- `SecondOrderDiscretizedFilter` : Explicit name of what it does intuitively.
- `LaplaceDissipationFilter` : Explicit name for robotics/signal-processing jargon
- <s>`SecondOrderVelocityDissipation`</s>